src/lib/repositories/impl/chef_repository_impl.py

```diff
@@ -26,3 +27,13 @@ def update_by_id(self, chef_id, chef):
         current_chef = self.get_by_id(chef_id)
         current_chef.name = chef.name or current_chef.name
         current_chef.chef_skills = chef.chef_skills or current_chef.chef_skills
+
+    def get_available_chefs(self):
+        chefs = self.get_all()
+        orders = self.order_repository.get_all()
```

Comment:
Don't you think it is better to create a method in the order repo that gets the assigned order by a chef?

Reply:
done